### DISCUSSION TITLE
Total download size become significantly larger after v0.23?

### DISCUSSION THREAD

**Discussion by MuTsunTsai** [pyodide/pyodide#3784] [Q&A]
In my app https://github.com/MuTsunTsai/fontfreeze, I noticed that as I upgrade from Pyodide v0.22 to v0.23, the total package download size becomes about 1.5x larger, which is a far greater difference than all previous upgrades I've experienced with Pyodide. I wonder why that is the case? I already have `fullStdLib: false`, and I load two packages, `brotli` and `fonttools`.

**Comment by rth** [comment]
Thanks for the report!

For a baseline, if you look at the REPL, which is also a minimal install, in the dev console,
 - [v0.23.1](https://pyodide.org/en/0.23.1/console.html) is 19.50 MB / 5.28 MB transferred
 - [v0.22.0](https://pyodide.org/en/0.22.0/console.html) is 14.64 MB / 6.34 MB transferred

we are compressing less the files ourselves, and relying on the CDN compression (as that overall works better https://github.com/pyodide/pyodide/pull/3667) . So it's expected that the first number increased, what matters is the second number corresponding to the actual transferred size with compression.

If I open https://mutsuntsai.github.io/fontfreeze/ (which currently uses v0.22)   in the dev console Firefox I see  `6.75 MB / 3.61 MB transferred` which does not make sense, particularly given the `pyodide.asm.wasm` file is not loaded, which does contain the interpreter and accounts for the size difference.  I also don't see the brotli and fonttools being loaded in the dev console.
So maybe Firefox dev console doesn't accurately report download sizes for downloads from a a webworker?

In Chrome I see, `7.7 MB transferred / 16.0 MB resources` (with cache disabled) which is more reasonable, and you shouldn't see an increase over that for the transferred size  for v0.23.

**Reply by MuTsunTsai** [reply]
I see, that makes sense. The numbers I'm getting are the uncompressed file sizes, so that explains it. Thanks.
